tests/Oryx.RuntimeImage.Tests/NodeImagesTest.cs

```diff
@@ -405,7 +406,7 @@ public async Task GeneratesScript_CanRun_AppInsightsModule_NotFound(string nodeV
             // Arrange
             var imageName = string.Concat("oryxdevms/node-", nodeVersion);
```

Comment:
As we discussed, could you add a comment or change the name of the test to make it clear about what we are testing here?

Reply:
sure